incubator/binding-kafka-grpc.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/grpc/schema/kafka.grpc.schema.patch.json

```diff
@@ -145,26 +124,41 @@
                                     ]
                                 }
                             },
-                            "with": false,
+                            "with":
+                            {
+                                "properties":
+                                {
+                                    "scheme":
+                                    {
+                                        "title": "Scheme",
+                                        "type": "string"
+                                    },
+                                    "authority":
+                                    {
+                                        "title": "Authority",
+                                        "type": "string"
+                                    }
+                                },
+                                "additionalProperties": false
+                            },
+                            "required":
+                            [
+                                "entry",
```

Comment:
`entry` should be removed here as `route` doesn't have an `entry` property (yet), right?